snapcraft/storeapi/__init__.py

```diff
@@ -296,7 +305,8 @@ def get_package(self, snap_name, channel, arch):
         }
         params = {
             'channel': channel,
-            'fields': 'status,download_url,download_sha512',
+            'fields': 'status,download_url,download_sha512,snap_id,'
+                      'developer_id,release',
```

Comment:
as commented above, "developer_id" is not what you are looking for for validation assertions, it's always signed by the publisher.